frontend/src/components/Breadcrumb.tsx

```diff
@@ -0,0 +1,45 @@
+import { FC } from 'react'
+import Link from 'next/link'
+
+type BreadcrumbItemType = {
+    text: string;
+    link: string;
+  };
+
+export interface BreadcrumbProps {
+    id?: string;
+    items?: BreadcrumbItemType[];
+}
+
+const Breadcrumb: FC<BreadcrumbProps> = ({ id, items }) => {
+  return (
+    <section className="container mx-auto p-4">
+    <nav aria-label="breadcrumbs" id={id}>
+      <ul className="block text-blue-link font-body">
```

Comment:
Consider adding the required canada.ca breadcrumb here, so it will always be rendered.